test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -23,7 +23,7 @@ public class StartTests : BaseE2ETest
         private const string _serverNotReady = "Host was not ready after 10 seconds";
         public StartTests(ITestOutputHelper output) : base(output) { }
 
-        [Fact]
+        [Fact(Skip = "Test broken due to ASP.NET Core message suppression not taking effect.")]
```

Comment:
Could you please open an issue for the follow-up investigation and reference it here? Thanks.